macros/sql_gen/gen_joined_metrics_cte.sql

```diff
@@ -139,11 +139,21 @@
     {%- else %}
         join_metrics__{{previous_cte_number}}.*
     {%- endif %}
-    {%- for metric in metric_tree.ordered_expression_set %}
+    {%- for metric in metric_tree.expression_set %}
         {%- if metric_tree.ordered_expression_set[metric] == cte_number %}
-        ,({{metrics_dictionary[metric].expression | replace(".metric_value","")}}) as {{metrics_dictionary[metric].name}}
+            {%- set metric_definition = metrics_dictionary[metric].expression %}
```

Comment:
```suggestion
            {%- set metric_expression = metrics_dictionary[metric].expression %}
```

I have not updated the rest of the places this impacts 😬

Reply:
Ah - this is a good callout. I've tended to use metric definition to refer to the entire object that exists in `metrics_dictionary` for a particular metric. So if this is just the single expression then I think renaming is a good call